c++/src/kj/async-inl.h

```diff
@@ -35,10 +35,10 @@
 #include <intrin.h>
 #endif
 
-KJ_BEGIN_HEADER
-
 #include <kj/list.h>
 
+KJ_BEGIN_HEADER
```

Comment:
Since async-inl.h is only ever included from async.h, I wonder if all of async-inl.h's includes are covered by async.h's KJ_BEGIN_HEADER anyway...

Reply:
Oh, I suppose the include of async-inl.h in async.h should really be after KJ_END_HEADER. Oh well.